src/foam/nanos/dig/DIG.js

```diff
@@ -37,7 +37,7 @@ foam.CLASS({
   ],
 
   imports: [
-    'nSpecDAO'
+    'AuthenticatedNSpecDAO'
```

Comment:
Isn't the nSpecDAO already authenticated?

Reply:
nope ..it's not